sdmetrics/reports/multi_table/quality_report.py

```diff
@@ -25,6 +27,26 @@ def __init__(self):
         self._properties_scores = {}
         self._is_generated = False
         self._package_version = None
+        self._property_errors = {}
+
+    def _print_results(self, out=sys.stdout):
+        """Print the quality report results."""
+        if pd.isna(self._overall_quality_score) & any(self._property_errors.values()):
+            out.write('\nOverall Quality Score: Error computing report.\n\n')
```

Comment:
Run this line whenever `_overall_quality_score` is `nan`, so we don't need `_property_errors`.